core/block.js

```diff
@@ -38,13 +38,13 @@ goog.requireType('Blockly.IASTNodeLocation');
  * Class for one block.
  * Not normally called directly, workspace.newBlock() is preferred.
  * @param {!Blockly.Workspace} workspace The block's workspace.
- * @param {?string} prototypeName Name of the language object containing
+ * @param {!string} prototypeName Name of the language object containing
```

Comment:
Changing this to non-nullable breaks compile test. 
```
found   : (null|string)
required: string
  502|   return new Blockly.Block(this, prototypeName, opt_id);
```

It is caused by `Blockly.Workspace.prototype.newBlock` having `prototypeName` be `{?string}`

Reply:
Thanks for the ping! I just figured it would work so I didn't watch the build hehe. Should be fixed now.